src/test/resources/unit/basic-test/upsert-job-np/test-overrides-plugin-config-prod.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~    Copyright 2018 Edmunds.com, Inc.
+  ~
+  ~        Licensed under the Apache License, Version 2.0 (the "License");
+  ~        you may not use this file except in compliance with the License.
+  ~        You may obtain a copy of the License at
+  ~
+  ~            http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~        Unless required by applicable law or agreed to in writing, software
+  ~        distributed under the License is distributed on an "AS IS" BASIS,
+  ~        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~        See the License for the specific language governing permissions and
+  ~        limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>unit-test-group</groupId>
+    <artifactId>unit-test-artifact</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.edmunds</groupId>
+                <artifactId>databricks-maven-plugin</artifactId>
+                <version>${project.version}</version>
+                <configuration>
+                    <environment>PROD</environment>
+                    <dbJobFile>databricks-job-settings.json</dbJobFile>
```

Comment:
Off topic: can this be different between prod and qa?

Reply:
Thanks. FYI, we have cases where we don't want to schedule QA jobs, so this would be helpful in a future release if you someone else has that use case.